master/buildbot/test/fake/fakedb.py

```diff
@@ -2322,7 +2322,11 @@ def getMaster(self, masterid):
         return defer.succeed(None)
 
     def getMasters(self):
-        return defer.succeed(sorted(self.masters.values()))
+        def masterKey(master):
+            return master['id']
+
+        return defer.succeed(sorted(self.masters.values(),
+                                    key=masterKey))
```

Comment:
Even though I disagree with you, I have changed this instance to use a lambda,
because I don't want this PR to be blocked on this.